app-crypt/trousers/trousers-0.3.14-r3.ebuild

```
@@ -64,5 +64,7 @@ src_install() {
 	systemd_dounit "${FILESDIR}"/tcsd.service
 	udev_dorules "${FILESDIR}"/61-trousers.rules
 	fowners tss:tss /var/lib/tpm
+	fowners root:tss /etc/tcsd.conf
```

```suggestion
	fowners :tss /etc/tcsd.conf
```
Same for other ebuild.